### ISSUE TITLE
Make pagination bar viewable in super admin

### ISSUE THREAD

**Issue by ecurrencyhodler** [stakwork/sphinx-tribes#1233]
**Describe the bug**
Right now I am unable to view the pagination bar at the bottom, even if I scroll down, unless I adjust the zoom. 

https://github.com/stakwork/sphinx-tribes/assets/32662508/85895488-53ac-4092-9ffc-3381e343863f

**Expected behavior**
I should be able to view the pagination bar at the bottom by scrolling down.

**Desktop (please complete the following information):**
 - Browser: Chrome

### Acceptance Criteria
- [ ] I have tested on Chrome desktop
- [ ] I have posted a video of me scrolling down to view the page
- [ ] Component Test: make sure pagination is visible

**Comment by hritikmunde** [comment]
@ecurrencyhodler , Can I work on this?

**Comment by ecurrencyhodler** [comment]
Yes absolutely. this is yours.

**Comment by ecurrencyhodler** [comment]
Did you sign up on our website yet? If so, what's your profile? https://community.sphinx.chat/bounties

**Comment by ecurrencyhodler** [comment]
Nvm found you!

**Comment by Vayras** [comment]
@ecurrencyhodler I Think This has been already addressed

**Comment by ecurrencyhodler** [comment]
Okay. We can turn this bounty into a unit test then.

**Comment by ecurrencyhodler** [comment]
@hritikmunde we are going to change this issue into a unit test only. Once #1223 has been merged, you can begin working on this bounty.

**Comment by hritikmunde** [comment]
@ecurrencyhodler , okay got it.

**Comment by ecurrencyhodler** [comment]
@hritikmunde this is now ready to be worked on. You can grab the latest from master.

**Comment by hritikmunde** [comment]
@ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.

**Comment by kevkevinpal** [comment]
> @ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.

I'm not seeing any reference to the `__test__` path in here https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/jest.config.json could you point out where exactly in the file it is pointing to that?

**Comment by hritikmunde** [comment]
> > @ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.
> 
> I'm not seeing any reference to the `__test__` path in here https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/jest.config.json could you point out where exactly in the file it is pointing to that?

I'm proposing to create a folder __test_ and create a new test file for pagination. What do you think?

**Comment by kevkevinpal** [comment]
> @ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.

I realized you were right here, there seems to be a second jest.config.json file that specifies the `__tests__` folder, I made some changes to move some tests into `__tests__` folder

**Comment by kevkevinpal** [comment]
> > > @ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.
> > 
> > 
> > I'm not seeing any reference to the `__test__` path in here https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/jest.config.json could you point out where exactly in the file it is pointing to that?
> 
> I'm proposing to create a folder __test_ and create a new test file for pagination. What do you think?

Yea I have considered that before but I think that it makes it easier to create and validate tests if they are closer to the files they test. Please read [this post](https://stackoverflow.com/questions/42385701/pros-and-cons-of-placing-test-files-in-the-same-folder-as-source-files-or-separ)  for the comparison to the two approaches but I think it is best practice to keep the tests near the code you are testing

**Comment by hritikmunde** [comment]
> > @ecurrencyhodler , the tests for superAdmin are in the file `TableComponent.Spec.tsx` which has the wrong naming convention and is not in the folder `__test__` which is where jest finds test cases according to the regex configured in file `jest.config.js`. To make a unit test for pagination I feel I should create a new file `Pagination.spec.ts` in the folder `frontend\app\src\pages\superadmin\tableComponent\__test__` and add the test case there.
> 
> 
> 
> I realized you were right here, there seems to be a second jest.config.json file that specifies the `__tests__` folder, I made some changes to move some tests into `__tests__` folder

Yes I have added tests to that particular file. It's working will make changes needed in review

**Comment by ecurrencyhodler** [comment]
Closed by #1288
